src/ParserConfigFile.cpp

```diff
@@ -405,6 +421,12 @@ void Raytracer::ParserConfigFile::parseConesInf(
       if (materialName == "reflective") {
         newConeInf->setMaterial(
             _factory.create<Raytracer::Reflections>("reflection"));
+      } else if (materialName == "transparent") {
+        newConeInf->setMaterial(
+            _factory.create<Raytracer::Transparency>("transparent"));
+      } else if (materialName == "refractive") {
+        newConeInf->setMaterial(
+            _factory.create<Raytracer::Refractions>("refraction"));
```

Comment:
```suggestion
      } else if (materialName == "refractive") {
        newConeInf->setMaterial(
            _factory.create<Raytracer::Refractions>("refraction"));
      } else if (materialName == "transparent") {
        newConeInf->setMaterial(
            _factory.create<Raytracer::Transparency>("transparent"));
```

pareil ici, j'ai juste inversé